src/clarity-angular/data/_tables.clarity.scss

```diff
@@ -2,39 +2,15 @@
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
 
-// Backgrounds
-$clr-table-bgcolor: $clr-white !default;
-$clr-thead-bgcolor: $clr-near-white !default;
-
-// Borders
-$clr-tablerow-bordercolor: $clr-light-gray !default;
-$clr-table-bordercolor: $clr-light-midtone-gray !default;
-$clr-table-borderwidth: $clr-default-borderwidth !default;
-$clr-table-borderstyle: $clr-table-borderwidth solid $clr-table-bordercolor !default;
-
-// Etc.
-$clr-table-fontsize: clr-getTypePropertyValueForDomElement(table_text, font-size);
-$clr-table-lineheight: 0.583333rem !default;
-$clr-table-cellpadding: 0.5rem !default;
-$clr-table-smallpadding: 0.25rem !default;
-
-$clr-table-lineheightPaddingShim: (1rem - $clr-table-lineheight)/2 !default;
-$clr-table-topcellpadding: $clr-table-cellpadding - $clr-rem-1px !default;
-$clr-table-bottomcellpadding: $clr-table-cellpadding - $clr-rem-1px !default;
-
-// Border radius of corner cells needs to be slightly less than the table's,
-// To make sure they correctly cover the whole area up to the border.
-$clr-table-cornercellradius: calc(#{$clr-default-borderradius} - 1px) !default;
-
 // Mixin for basic table styles to be able to reuse them on non-table elements.
 @mixin basic-table($table, $thead, $tbody, $tr, $th, $td) {
     #{$table} {
         border-collapse: separate;
         border: $clr-table-borderstyle;
-        border-radius: $clr-default-borderradius;
+        border-radius: $clr-global-borderradius;
 
         background-color: $clr-table-bgcolor;
-        color: clr-getTextColor($clr-table-bgcolor, text);
+        color: $clr-table-font-color;
         margin: 0;
         margin-top: 1rem;
```

Comment:
Yes I am definitely not sure about variables keeping half variables in the file and half outside. May be we should just move size variables in the file and keep the color variables outside? This can be addressed later on but we need to decide what to do.